variables.tf

```diff
@@ -39,16 +39,16 @@ variable "additional_ses_sendemail_account_ids" {
   default     = []
 }
 
-variable "aws_region" {
+variable "api_gateway_zone_id" {
   type        = string
-  description = "The AWS region to communicate with."
-  default     = "us-east-1"
+  description = "The Route 53 hosted zone ID for us-east-1 endpoints."
```

Comment:
The description shouldn't mention `us-east-1`, as @dav3r pointed out [here](https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/92#discussion_r1481987693).

Reply:
See commit b56552c9fc3074bb725b54bb94372b601cb00a0f.